### PR TITLE
updated jenkins job status badge in README

### PR DESCRIPTION
**Pull request by sivanaikk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
+[![Performance Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520catalogue%2520%28v5.5.0%29%2520pipeline%2F&label=performance%20tests)](https://jenkins.iudx.io/view/iudx-v5.5.0/job/iudx%20catalogue%20(v5.5.0)%20pipeline/lastBuild/performance/)
+[![Security Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520catalogue%2520%28v5.5.0%29%2520pipeline%2F&label=security%20tests)](https://jenkins.iudx.io/view/iudx-v5.5.0/job/iudx%20catalogue%20(v5.5.0)%20pipeline/lastBuild/zap/)
+[![Integration Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520catalogue%2520%28v5.5.0%29%2520pipeline%2F&label=integration%20tests)](https://jenkins.iudx.io/view/iudx-v5.5.0/job/iudx%20catalogue%20(v5.5.0)%20pipeline/lastBuild/Integration_20Test_20Report/)
```
Comment (iSRIDHARRAO) [inline README.md:7]: Integration tests report are available within unit tests report and use single badge for both. Please check

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-[![Performance Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520catalogue%2520%28master%29%2520pipeline%2F&label=performance%20tests)](https://jenkins.iudx.io/job/iudx%20catalogue%20(master)%20pipeline/lastBuild/performance/)
-[![Security Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520catalogue%2520%28master%29%2520pipeline%2F&label=security%20tests)](https://jenkins.iudx.io/job/iudx%20catalogue%20(master)%20pipeline/lastBuild/zap/)
-[![Integration Tests](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fiudx%2520catalogue%2520%28master%29%2520pipeline%2F&label=integration%20tests)](https://jenkins.iudx.io/job/iudx%20catalogue%20(master)%20pipeline/lastBuild/Integration_20Test_20Report/)
+[![Build Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fview%2Fiudx-v5.5.0%2Fjob%2Fiudx%2520catalogue%2520%28v5.5.0%29%2520pipeline%2F)](https://jenkins.iudx.io/view/iudx-v5.5.0/job/iudx%20catalogue%20(v5.5.0)%20pipeline/)
```
Comment (karun-singh) [inline README.md:2]: This badge should be linked to the last build's build status.

**Review by karun-singh — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (sivanaikk) [reply README.md]: updated to show last build

File: README.md (hunk 1 above)
Comment (sivanaikk) [reply README.md]: added single badge for integration and unit tests

**Pull request merged**